filebeat/docs/migration.asciidoc

```diff
@@ -117,8 +117,8 @@ filebeat:
     # Each - is a prospector. Below are the prospector specific configurations
     -
       paths:
-        - /var/log/messages
-        - "/var/log/*.log"
+        - /system/var/log/messages
+        - "/system/var/log/*.log"
```

Comment:
Not sure I understand the changes here. The `/system/` part doesn't show up in the initial LSF config, and `/var/log` is more standard.